entity-framework/core/extensions/index.md

```diff
@@ -211,3 +211,11 @@ LINQ extensions to Entity Framework Core 3.1 to support Microsoft SQL Server Tem
 Adds hierarchyid support to the SQL Server EF Core provider. For EF Core: 3.
 
 [GitHub repository](https://github.com/efcore/EFCore.SqlServer.HierarchyId)
+
+### linq2db.EntityFrameworkCore
+
+Alternative translator of LINQ queries to SQL expressions.
+
+Includes support for advanced SQL features such as CTEs, bulk copy, table hints, windowed functions, temporary tables, and database-side create/update/delete operations.
```

Comment:
Pls list supported EF Core versions like this: "For EF Core: 2, 3."

Reply:
Done